neo4j_utils/_n4jversion.py

```diff
@@ -13,8 +14,13 @@ class Neo4jVersion:
     def __init__(self):
         try:
             cmd = ['neo4j-admin', '--version']
-            version = subprocess.check_output(cmd).decode().strip()
+            output = subprocess.check_output(cmd).decode().strip()
+            semver_pattern = r'\b\d+\.\d+\.\d+\b'
+            matches = re.findall(semver_pattern, output)
+            version = matches[0]
```

Comment:
Hello @nilskre , thanks for this fix! Though this line will throw `IndexError`, and also the next line should be updated accordingly. If you address this I'll merge the PR right away

Reply:
Just so you know, it is now adapted accordingly.

Reply:
I see we are in a try except block already. Then it could be:

```
if version := re.findall(semver_pattern, output):
	version = version[0]
else:
	raise RuntimeError(f'Failed to state Neo4j version from command output: {output}')
```

Reply:
And maybe at the end, instead of raising the error, it could emit a warning and fallback to the current stable Neo4j version (`5`)

Reply:
Now adapted based on your feedback @deeenes